spec/unit/devtools/flay/scale/measure_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'flay'
+
+module Devtools
+  module Flay
+    describe Scale, '#measure' do
+      subject(:measure) { instance.measure }
+
+      let(:minimum_mass) { 0        }
+      let(:files)        { [double] }
+      let(:instance) do
+        described_class.new(minimum_mass: minimum_mass, files: files)
+      end
```

Comment:
I personally prefer having the "multinline let blocks" separated from the "single line let blocks via a white line, and each multiline let block separated from others with a white line one also.